src/Nethermind/Nethermind.Merge.Plugin/Data/V1/ExecutionPayloadV1.cs

```diff
@@ -32,18 +34,6 @@ namespace Nethermind.Merge.Plugin.Data.V1
     /// </summary>
     public class ExecutionPayloadV1
```

Comment:
Withdrawals should be implemented as V2, so you need new methods exposed in EngineAPI

Reply:
Given the current implementation logic, having a separate `V2` is redundant. I either drop `V1` or change it to `V2`. Otherwise, the class structure needs to be changed a lot which, IMO, is over-complication.

Reply:
However, I am talking about RPC methods, engine_newPayloadV2, engine_forkchoiceUpdatedV2...